crates/bevy_ecs/src/query/fetch.rs

```diff
@@ -1607,6 +1608,22 @@ macro_rules! impl_anytuple_fetch {
 all_tuples!(impl_tuple_fetch, 0, 15, F, S);
 all_tuples!(impl_anytuple_fetch, 0, 15, F, S);
 
+pub struct NopWorldQuery<Q: WorldQuery>(PhantomData<Q>);
```

Comment:
Please add `#[doc(hidden)]` on this.

Reply:
I dont see why, it might be useful to someone. Probably should have doc comments explaining what it is though.